Extending/Packages/Creating-a-nuget-package/index.md

```diff
@@ -1,13 +1,14 @@
 ---
 versionFrom: 8.0.0
+versionTo: 8.0.0
 meta.Title: "Creating a Nuget version of an Umbraco package"
 meta.Description: "A guide to creating a Nuget version of an Umbraco package"
 ---
 
 # Creating a NuGet version of a package
 
 :::note
-This tutorial is for Umbraco 8 - however, a lot of the things covered here will be the same or similar in Umbraco 7. Regarding Umbraco 9, check the instructions in [Creating a Package article](../Creating-a-Package/index.md).
+This tutorial is for Umbraco 8 - however, a lot of the things covered here will be the same or similar in Umbraco 7. For  Umbraco 9 or higher, check the instructions in [Creating a Package article](../Creating-a-Package/index.md).
```

Comment:
I guess this should link to the new platform as it states that it is for Umbraco 9 and later 🤔

Reply:
Umm... For Umbraco 9 or later, it's linked to the "Creating a Package" article which is different from this one - "Creating a nuget package" 😀 We do not have an equivalent article for Creating a NuGet package on v9 +

Reply:
On the new platform in the creating a package article it also show how you can create a NuGet package:
- https://docs.umbraco.com/umbraco-cms/extending/packages/creating-a-package#creating-a-nuget-package 

Which I think is to where it should link to, as the article you are linking to is only for v8, and then the text is wrong when it says: _"For  Umbraco 9 or higher, check the instructions in [Creating a Package article]"_ I hope it makes sense :D

Reply:
Ah, I see. I couldn't find that part in the new doc site so thought it was not there. I'll fix it :D

Reply:
Awesome :D